content/Coalesce.Starter.Vue.Web/Coalesce.Starter.Vue.Web.csproj

```diff
@@ -10,6 +10,11 @@
     <Content Update="Views\**\*;appsettings.json;web.config">
       <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
     </Content>
+    <Content Update="Properties\launchSettings.json">
+      <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </Content>
```

Comment:
This doesn't need to be added to the csproj.

Reply:
Foiled again by Rider.